roles/openshift_examples/tasks/stream_secrets.yml

```diff
@@ -31,4 +31,10 @@
 - name: Modify registry paths if registry_url is not registry.redhat.io
   shell: >
     find {{ examples_base }} -type f | xargs -n 1 sed -i 's|registry.redhat.io|{{ registry_host | quote }}|g'
-  when: registry_host != '' and openshift_examples_modify_imagestreams | default(False) | bool
+  when: openshift_examples_registryurl.split('/') | length > 2 and registry_host != '' and openshift_examples_modify_imagestreams | default(False) | bool
+
+- name: Modify registry paths if registry_url is not registry.redhat.io and using Satellite
+  shell: >
+    find {{ examples_base }} -type f | xargs -n 1 sed -i -e 's|registry.redhat.io/\([^/]*\)/\(.*\)$|registry.redhat.io/\1_\2|g' \
+         -e 's/openshift3_ose-//g' -e 's/openshift3-ose-//g' -e 's|registry.redhat.io/\([^/]*\)$|{{ satellite_prefix | quote }}\1|g'
+  when: openshift_examples_registryurl.split('/') | length == 2 and satellite_prefix != '' and openshift_examples_modify_imagestreams | default(False) | bool
```

Comment:
Please use multi-line 'and' conditionals.

Reply:
I've modified as you mentioned. Thanks !